themes/vague.json

```
@@ -2,13 +2,21 @@
   "$schema": "https://zed.dev/schema/themes/v0.2.0.json",
   "name": "Vague",
   "author": "Aejkatappaja, L1NC5 (original vim theme by vague2k)",
+  "accents": [
+    "#c48282",
+    "#e8b589",
+    "#e0a363",
+    "#c3c3d5",
+    "aeaed1",
+    "#bb9bdb",
+    "#9010b5"
```

```suggestion
    "#6e94b2"
```
Let's avoid using a custom color here and stick to the theme palette